app/Console/Commands/FlightawareImport.php

```diff
@@ -0,0 +1,232 @@
+
+        $pbar = $this->output->createProgressBar($total_steps);
+        $pbar->setFormat('custom');
+
+        $flights_count = 0;
+
+        for ($y = 0; $y < $chunk_count; $y++) {
+            $chunk = $chunks[$y];
+
+            $pbar->setMessage('Departures, chunk #' . $y);
+
+            $flights = [];
+
+            for ($i = 0; $i < intval(Config::get('flightaware.departure_pages_per_chunk')); $i++) {
+                if ($flights_count > $ratelimit_max_flights) {
+                    $pbar->setMessage('Ratelimit timeout');
+                    $pbar->advance();
+                    $flights_count = 0;
+                    sleep($ratelimit_timeout);
+                }
+
+                $url = Config::get('flightaware.base') . '/schedules/' . $chunk[0]->format('Y-m-d\TH:i:s') . '/' . $chunk[1]->format('Y-m-d\TH:i:s') . '?origin=KATL';
+                $pbar->setMessage($url);
+
+                if (!Config::get('flightaware.dryrun')) {
+                    $res = $client->request('GET', $url);
+                    $data = json_decode($res->getBody(), true);
+
+                    $flights = array_merge($flights, $data['scheduled']);
+                    $flights_count += sizeof($data['scheduled']);
+                }
+
+                $pbar->advance();
+            }
+
+            $pbar->setMessage('Arrivals, chunk #' . $y);
+
+            for ($i = 0; $i < intval(Config::get('flightaware.arrival_pages_per_chunk')); $i++) {
+                if ($flights_count > $ratelimit_max_flights) {
+                    $pbar->setMessage('Ratelimit timeout');
+                    $pbar->advance();
+                    $flights_count = 0;
+                    sleep($ratelimit_timeout);
+                }
+
+                $url = Config::get('flightaware.base') . '/schedules/' . $chunk[0]->toIso8601String() . '/' . $chunk[1]->toIso8601String() . '?destination=KATL';
+                $pbar->setMessage($url);
+
+                if (!Config::get('flightaware.dryrun')) {
+                    $res = $client->request('GET', $url);
+                    $data = json_decode($res->getBody(), true);
+
+                    $flights = array_merge($flights, $data['scheduled']);
+                    $flights_count += sizeof($data['scheduled']);
+                }
+
+                $pbar->advance();
+            }
+
+            $pbar->setMessage('Saving flights for chunk #' . $y);
+
+            for ($i = 0; $i < sizeof($flights); $i++) {
+                $flight = $flights[$i];
+                $pbar->setMessage('Saving ' . $flight['ident']);
+
+                $callsign = $flight['ident'];
+
+                $existing = RealopsFlight::where('flight_number', $callsign)->first();
+                if ($existing !== null) {
+                    continue;
+                } // this event has already been processed
+
+                // insert
+                $new_flight = new RealopsFlight;
+
+                // parse the departure and arrival times
```

Comment:
To clean this up, let's remove these comments. If it's not as clear by the code, we can split to a separate function to name the action. But these actions explain themselves IMO

Reply:
Done 0971fe0